src/errors.rs

```diff
@@ -1,4 +1,7 @@
-/// Hashing errors.
+/// # Fantom/libHash/errors
+///
+/// This file defines an error variant which can be used for the Hash trait.
```

Comment:
A nice link could be given with
```
[`Hash`]
```

Reply:
Also I forgot to mention that module comments should start with `//!` rather than with `///` in order to be displayed correctly in the output of `cargo doc`.